.github/workflows/github_actions_aws_rhel_python64.yml

```diff
@@ -26,11 +26,11 @@ jobs:
     timeout-minutes: 40
     strategy:
       matrix:
-        module_name: ["nidmm", "niswitch"]
+        module_name: ["nidcpower", "nidmm", "nifgen", "niscope", "niswitch"]
```

Comment:
`nitclk`?

Reply:
I can add it.

`nitclk` system tests use NI-SCOPE runtime. There is a bug in in the latest release of NI-SCOPE that causes the `niscope` and `nitclk` system tests to fail. The bug has been fixed since then.

We have two options:
1) Wait for the next release of NI-SCOPE (in 2 months time) before enabling `nitclk` and `niscope` tests
2) Install the latest dev build of NI-SCOPE runtime on `nimibot` and enable the tests now.

My preference is to wait. @marcoskirsch, any objections?